index.html

```diff
@@ -8,14 +8,76 @@
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <link rel="stylesheet" href="/home/coder/Documents/GitHub/HTML-CSS-Coursework-Week2/css/style.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <header>
+        <img id="logo" src="img/karma-logo.svg"  alt="karma logo">
+        <nav>
+            <ul>
+                <li id="meetKarma"> <a href="#"> Meet Karma </a> </li>
+                <li> <a href="#"> How it Works </a> </li>
+                <li> <a href="#"> Store </a> </li>
+                <li> <a href="#"> Blog </a> </li>
+                <li> <a href="#"> Help </a> </li>
+                <li> <a href="#"> Login </a> </li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <article class="intro">
+            <p> Introducing Karma </p>
+            <p id="firstText">Bring WiFi with you, everywhere you go.</p>
+            <button id="learnMore"> Learn More </button>             
+        </article>
+       
+        <article class="part-one">           
+            <p>Everyone needs a little Karma</p>
+        </article>
+       
+        <section class="contents">     
+            <article id="content-1">
+                
+                <img src="img/icon-devices.svg" alt="Animated picture of devices"/>
+                <a href="#"> <p>Internet for all devices</p> </a>
+                
+            </article>
+            
+            <article id="content-2">
+                
+                <img src="img/icon-coffee.svg" alt="Animated coffee cup"/>
+                <a href="#"> <p>Boost your productivity</p> </a>
+            
+            </article>
+            
+            <article id="content-3"> 
+                
+                <img src="img/icon-refill.svg" alt="Animated petrol pump">
+                <a href="#"> <p>Pay as You Go</p> </a>
+                
+            </article>
+         </section>
+    </main>
+
+    <footer>
+        <p>Join us on</p>
+        
+        <ul id="iconsWrapper">
+            <li><a href="#"><img id="icon" src="img/twitter-icon.svg" alt="Twitter Icon"></a></li>
+            <li><a href="#"><img id="icon" src="img/facebook-icon.svg" alt="Facebook Icon"></a></li>
+            <li><a href="#"><img id="icon" src="img/instagram-icon.svg" alt="Instagram Icon"></a></li>
+        </ul>
```

Comment:
Good semantic HTML here.
Again, like everywhere else, don't use an ID. Change it to a class: `.icon`